src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -14,30 +14,89 @@
 
 package com.googleinterns.zoomtube.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import javax.servlet.ServletException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/**
- * Provides information on a lecture.
- */
+/** Provides information on a lecture. */
 @WebServlet("/lecture")
 public class LectureServlet extends HttpServlet {
+  /* Used to generate a Pattern for a Video URL. */
+  private static final String PATTERN =
```

Comment:
Maybe call it like YOUTUBE_VIDEO_URL_PATTERN?

Reply:
fixed